### PR TITLE
rclone-browser: Add version 1.8.0

### PR DESCRIPTION
**Pull request by erogy**
https://github.com/kapitainsky/RcloneBrowser

### CODE REVIEW

File: bucket/rclone-browser.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "innosetup": true,
+    "pre_install": [
+        "if (Test-Path \"$persist_dir\\RcloneBrowser.ini\") {",
+        "   Copy-Item \"$persist_dir\\RcloneBrowser.ini\" \"$dir\"",
+        "} else {",
+        "   Set-Content \"$dir\\RcloneBrowser.ini\" @('[Settings]', 'checkRcloneBrowserUpdates=false', 'checkRcloneUpdates=false') -Encoding Ascii",
+        "}",
+        "if (!(Test-Path \"$persist_dir\\tasks.bin\")) {",
+        "   New-Item \"$dir\\tasks.bin\" | Out-Null",
+        "}"
```
Comment (Ash258) [inline bucket/rclone-browser.json:26]:
```suggestion
        "if (!(Test-Path \"$persist_dir\\tasks.bin\")) { New-Item \"$dir\\tasks.bin\" | Out-Null }",
        "if (Test-Path \"$persist_dir\\RcloneBrowser.ini\") {",
        "   Copy-Item \"$persist_dir\\RcloneBrowser.ini\" \"$dir\"",
        "} else {",
        "   $forwardPer = \"$persist_dir\\downloads\" -replace '\\\\', '/'",
        "   Set-Content \"$dir\\RcloneBrowser.ini\" @(",
        "       '[Settings]', 'checkRcloneBrowserUpdates=false', 'checkRcloneUpdates=false', \"defaultDownloadDir=$forwardPer\"",
        "   ) -Encoding Ascii",
        "}"
    ],
```

File: bucket/rclone-browser.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    "uninstaller": {
+        "script": "Copy-Item \"$dir\\RcloneBrowser.ini\" \"$persist_dir\" -ErrorAction SilentlyContinue -Force"
+    },
+    "shortcuts": [
+        [
+            "RcloneBrowser.exe",
+            "Rclone Browser"
+        ]
+    ],
+    "persist": "tasks.bin",
```
Comment (Ash258) [inline bucket/rclone-browser.json:37]: Does it use QT's atomic saving for tasks.bin file?

```suggestion
    "persist": [
        "downloads",
        "tasks.bin"
    ],
```

Comment (erogy) [reply bucket/rclone-browser.json]: Not sure what it is but the tasks.bin file isn't recreated every time it's saved unlike the .ini file.

**Pull request merged**